Doesn't work with scoped modules

`var log = require('@zetta/log-colors');` doesn't work. the error reference lists only the scope and not the scoped module

```
⠋ Installing @zetta
npm ERR! addLocal Could not install /Users/tonyarcher/code/zetta/sql-boilerplate/@zetta
npm ERR! Darwin 15.6.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install" "@zetta" "--save"
npm ERR! node v6.3.0
npm ERR! npm  v3.10.5
npm ERR! path /Users/tonyarcher/code/zetta/sql-boilerplate/@zetta
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall open

npm ERR! enoent ENOENT: no such file or directory, open '/Users/tonyarcher/code/zetta/sql-boilerplate/@zetta'
npm ERR! enoent ENOENT: no such file or directory, open '/Users/tonyarcher/code/zetta/sql-boilerplate/@zetta'
npm ERR! enoent This is most likely not a problem with npm itself
npm ERR! enoent and is related to npm not being able to find a file.
npm ERR! enoent 

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/tonyarcher/code/zetta/sql-boilerplate/npm-debug.log

⚠ @zetta installation failed
```

--- a/example/index.js
+++ b/example/index.js
@@ -10,3 +10,6 @@ const fs = require('fs');
 // Require files
 const helpers = require('../src/helpers');
 
+// Require scoped modules
+const autoInstall = require('@siddharthkp/auto-install');
+

--- a/src/helpers.js
+++ b/src/helpers.js
@@ -246,7 +246,10 @@ let removeLocalFiles = (modules) => modules.filter((module) => !module.name.incl
  */
 
 let removeFilePaths = (modules) => {
-    for (let module of modules) module.name = module.name.split('/')[0];
+    for (let module of modules) {
+        let slicedName = module.name.split('/')[0];
+        if (slicedName.substr(0, 1) !== '@') module.name = slicedName;
+    }
     return modules;
 };
 

--- a/test/testData.js
+++ b/test/testData.js
@@ -9,7 +9,8 @@ let usedModules = [
     {name: 'chokidar', dev: false},
     {name: 'yargs', dev: false},
     {name: 'fs', dev: false},
-    {name: '../src/helpers', dev: false}
+    {name: '../src/helpers', dev: false},
+    {name: '@siddharthkp/auto-install', dev: false}
 ];
 
 let unusedModules = [
@@ -19,13 +20,15 @@ let unusedModules = [
 
 let modulesNotInstalled = [
     {name: 'chai', dev: true},
-    {name: 'chokidar', dev: false}
+    {name: 'chokidar', dev: false},
+    {name: '@siddharthkp/auto-install', dev: false}
 ];
 
 let filteredUsedModules = [
     {name: 'chai', dev: true},
     {name: 'chokidar', dev: false},
-    {name: 'yargs', dev: false}
+    {name: 'yargs', dev: false},
+    {name: '@siddharthkp/auto-install', dev: false}
 ];
 
 module.exports = {

